### ISSUE TITLE
Figure out what to do with Euro sign translation in `deroff.rs`

### ISSUE THREAD

**Issue by necaris** [rust-dc/fish-manpage-completions#16]
The code in `deroff.py` translates the "Eu" and "eu" specifications to octal `244`, which is "&#164;" (general currency symbol) -- however, since the "Do" and "ct" specifications translate to "$" and "ct" respectively, it would seem that "&#8364;" is actually more appropriate. Thoughts?

**Comment by scooter-dangle** [comment]
Yeah, the actual `€` seems way more appropriate but maybe the original `deroff` has a reason for not using chars outside of Latin-1? Something to investigate more closely when we're able to do full completion generation to see if that messes with the shell completions.

**Comment by PureW** [comment]
Aye, should probably first do a translation to rust that mimicks the pyhton-script completely. That way we can verify functionality with python before improving or changing the behaviour.

**Comment by necaris** [comment]
Opened #20 to revert to the Python script behavior until we know what we want to change it to :smile:
